spec/requests/user/user_settings_spec.rb

```diff
@@ -238,14 +238,9 @@
   describe "PUT /update/:id" do
     before { sign_in user }
 
-    it "updates summary" do
```

Comment:
The profile tab no longer posts to this controller so we can remove the spec from here. That it still worked at all was more of side-effect of the delegation than an actual feature.